lib/iex/test/iex/helpers_test.exs

```diff
@@ -742,24 +785,28 @@ defmodule IEx.HelpersTest do
   describe "l" do
     test "loads a given module" do
       assert_raise UndefinedFunctionError, ~r"function Sample.run/0 is undefined", fn ->
-        Sample.run
+        Sample.run()
       end
 
       assert l(:nonexistent_module) == {:error, :nofile}
 
       filename = "sample.ex"
-      with_file filename, test_module_code(), fn ->
+
+      with_file(filename, test_module_code(), fn ->
         assert c(filename, ".") == [Sample]
-        assert Sample.run == :run
+        assert Sample.run() == :run
 
-        File.write! filename, "defmodule Sample do end"
-        elixirc ["sample.ex"]
+        File.write!(filename, "defmodule Sample do end")
+        elixirc(["sample.ex"])
 
         assert l(Sample) == {:module, Sample}
-        assert_raise UndefinedFunctionError, "function Sample.run/0 is undefined or private", fn ->
-          Sample.run
-        end
-      end
+
+        assert_raise UndefinedFunctionError,
+                     "function Sample.run/0 is undefined or private",
```

Comment:
Please extract the message to a variable.

Reply:
Good catch, it has been fixed.